### ISSUE TITLE
training script for any tensorflow based pre-trained model for question-answer

### ISSUE THREAD

**Issue by Akashdesarda** [huggingface/transformers#4275]
# ❓ Questions & Help

I am trying to train a question-answer model from scratch using TensorFlow backend. But I am getting unexpected result. It would be a great help if Hugging Face team or anyone can share training script of any TensorFlow based pre-trained model for question-answer (like 'bert-large-uncased-whole-word-masking-finetuned-squad' or 'distilbert-base-uncased-distilled-squad', etc) 

**A link to original question on Stack Overflow**:
https://stackoverflow.com/questions/61708227/how-to-train-any-hugging-face-transformer-model-eg-distilbert-for-question-ans
